docs/web3.eth.rst

```diff
@@ -617,7 +617,8 @@ The following methods are available on the ``web3.eth`` namespace.
     ``eth_getTransactionByBlockNumberAndIndex`` if ``block_identifier`` is an
     integer or one of the predefined block parameters ``'latest', 'earliest',
     'pending'``, otherwise delegates to
-    ``eth_getTransactionByBlockHashAndIndex``. If the transaction has not yet been mined throws :class:`web3.exceptions.TransactionNotFound`.
+    ``eth_getTransactionByBlockHashAndIndex``.
+    If a transaction is not found at specified arguments, throws :class:`web3.exceptions.TransactionNotFound`.
```

Comment:
This change is important because `get_transaction_by_block('pending', txn_index)` will return a transaction.